meson.build

```diff
@@ -1,7 +1,7 @@
 # SPDX-FileCopyrightText: 2008-2021 HPDCS Group <rootsim@googlegroups.com>
 # SPDX-License-Identifier: GPL-3.0-only
 
-project('ROOT-Sim', ['c', 'cpp'], version : '3.0.0',
+project('ROOT-Sim', ['c', 'cpp'], version : '3.0.1',
```

Comment:
We use semver for alpha releases, but so far in the projects we have condensed all alpha/beta releases in the x.x.0 version number.
3.0.1 should be read to a hotfix to a stable version.
If we want to version alphas, the rule should be:
`1.0.0-alpha < 1.0.0-alpha.1 < 1.0.0-alpha.beta < 1.0.0-beta < 1.0.0-beta.2 < 1.0.0-beta.11 < 1.0.0-rc.1 < 1.0.0.`